packages/x-charts/src/ChartsTooltip/ChartsItemTooltipContent.tsx

```diff
@@ -68,30 +68,7 @@ ChartsItemTooltipContent.propTypes = {
     }),
     series: PropTypes.shape({
       color: PropTypes.string,
-      data: PropTypes.oneOfType([
-        PropTypes.arrayOf(PropTypes.number),
-        PropTypes.arrayOf(PropTypes.number),
-        PropTypes.arrayOf(
-          PropTypes.shape({
-            id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
-            x: PropTypes.number.isRequired,
-            y: PropTypes.number.isRequired,
-          }),
-        ),
-        PropTypes.arrayOf(
-          PropTypes.shape({
-            color: PropTypes.string.isRequired,
-            endAngle: PropTypes.number.isRequired,
-            formattedValue: PropTypes.string.isRequired,
-            id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
-            index: PropTypes.number.isRequired,
-            label: PropTypes.string,
-            padAngle: PropTypes.number.isRequired,
-            startAngle: PropTypes.number.isRequired,
-            value: PropTypes.number.isRequired,
-          }),
-        ),
-      ]).isRequired,
+      data: PropTypes.arrayOf(PropTypes.number).isRequired,
```

Comment:
WDYT @alexfauquette @JCQuintas, is it a thing that could be a "BC" for people?
If you don't want this change, then we can document that in order to release v6 one needs to have `pnpm` installed regularly (not only with corepack).

Reply:
Can't we make so the monorepo calls whatever command it was used to start it with? 

eg:

`yarn buildTypes` => `yarn tsc`
`pnpm buildTypes` => `pnpm tsc`

Reply:
Probably can, but at this point, it's more or less too late—no longer worth it. 🙈